src/Stan/Inspection/Partial.hs

```diff
@@ -95,55 +103,36 @@ stan0004 = mkPartialInspectionList (Id "STAN-0004") (mkBaseListMeta "last")
 
 -- | 'Inspection' for 'stan0005' — partial 'GHC.List.!!' @STAN-0005@.
 stan0005 :: Inspection
-stan0005 = mkPartialInspection (Id "STAN-0005") (mkBaseListMeta "!!")
+stan0005 = mkPartialInspectionList (Id "STAN-0005") (mkBaseListMeta "!!")
+    & solutionL .~ []
 
 -- | 'Inspection' for 'stan0006' — partial 'GHC.List.cycle' @STAN-0006@.
 stan0006 :: Inspection
 stan0006 = mkPartialInspectionList (Id "STAN-0006") (mkBaseListMeta "cycle")
 
 -- | 'Inspection' for 'stan0007' — partial 'Data.OldList.genericIndex' @STAN-0007@.
 stan0007 :: Inspection
-stan0007 = mkPartialInspection (Id "STAN-0007") (mkBaseOldListMeta "genericIndex")
+stan0007 = mkPartialInspectionList (Id "STAN-0007") (mkBaseOldListMeta "genericIndex")
+    & solutionL .~ []
 
 -- | 'Inspection' for 'stan0008' — partial 'Data.Maybe.fromJust' @STAN-0008@.
 stan0008 :: Inspection
-stan0008 = Inspection
-    { inspectionId = Id "STAN-0008"
-    , inspectionName = "Partial: " <> nameMetaPackage <> "/" <> nameMetaName
-    , inspectionDescription = "Usage of partial function '" <> nameMetaName <> "' for 'Maybe'"
-    , inspectionSolution =
+stan0008 = mkPartialInspection (Id "STAN-0008") fromJustNameMeta "'Maybe'"
+    & solutionL .~
         [ "Use explicit pattern-matching over Maybe"
         , "Use one of the standard functions: 'maybe', 'fromMaybe'"
         ]
-    , inspectionCategory = one partial
-    , inspectionSeverity = Warning
-    , inspectionAnalysis = FindName nameMeta
-    }
   where
-    nameMeta :: NameMeta
-    nameMeta@NameMeta{..} = NameMeta
-        { nameMetaName       = "fromJust"
-        , nameMetaModuleName = "Data.Maybe"
-        , nameMetaPackage    = "base"
-        }
+    fromJustNameMeta :: NameMeta
+    fromJustNameMeta = set moduleNameL "Data.Maybe" $ mkBaseMeta "fromJust"
 
--- | 'Inspection' for 'stan0008' — partial 'Data.Maybe.fromJust' @STAN-0008@.
+-- | 'Inspection' for 'stan0009' — partial 'Text.Read.read' @STAN-0009@.
 stan0009 :: Inspection
-stan0009 = Inspection
-    { inspectionId = Id "STAN-0009"
-    , inspectionName = "Partial: " <> nameMetaPackage <> "/" <> nameMetaName
-    , inspectionDescription = "Usage of partial parsing function '" <> nameMetaName <> "'"
-    , inspectionSolution =
+stan0009 = mkPartialInspection (Id "STAN-0009") readNameMeta ""
+    & descriptionL .~ "Usage of partial parsing function 'read'"
```

Comment:
Maybe move this to function

```haskell
usage funName = "Usage of partial parsing function '" <> funName <> "'"

So it would be consistent around partial functions :slightly_smiling_face: